Assets/VRTK/Scripts/Controls/3D/VRTK_Spring_Lever.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Lever|Controls3D|0071
+using UnityEngine;
+
+namespace VRTK {
```

Comment:
brackets should be on their own line

```
x
{

}
```

(this will need to be done all throughout the file, if you're using Visual Studio you can auto format with `Ctrl+K` then `Ctrl+D`)